lib/src/widgets/board.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import '../config/app_spacing.dart';
+
+import './entry.dart';
+
+class Board extends StatelessWidget {
+  const Board({required this.data, Key? key}) : super(key: key);
+
+  final List data;
+  final double cacheExtent = 100.00;
+  final int indexCountStart = 1;
+  final double seperatorHeight = 16;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView.separated(
+      padding: const EdgeInsets.symmetric(horizontal: AppSpacing.xl),
+      itemCount: data.length,
+      addAutomaticKeepAlives: false,
+      cacheExtent: cacheExtent,
```

Comment:
weißt du, was genau cacheExtent bewirkt? Warum siehst du es als nötig an, das zu benutzen?

Reply:
So wie ich es verstanden hab ist das der Bereich vor und nach dem sichtbaren Bereich in dem die Elemente schon existieren damit nicht immer alle Elemente einer Liste geladen werden müssen. Wenn die Liste länger wird sollte das der Performance sehr gut tuen. Und man hat einen gewissen Puffer das die Elemente nicht erst geladen werden wenn sie wirklich im sichtbaren Bereich sind.